bot/utils/database-helper.js

```diff
@@ -51,4 +44,40 @@ module.exports.openDatabase = () => {
  */
 module.exports.getDatabase = () => {
     return database;
-};
\ No newline at end of file
+};
+
+/**
+ * Run all database migrations so that we reach the requested version.
+ * @param {string|undefined} version Which version to migrate to. If undefined, the latest version is chosen.
+ * @param {boolean} allowDown If true, allow downgrading database version. Otherwise (default), we throw an error.
+ */
+module.exports.doDatabaseMigrations = async (version = undefined, allowDown = false) => {
+    if (!version) {
```

Comment:
For consistency, I think this should be `if(version === undefined)`? Maybe we could add an eslint rule for that?

Reply:
I changed it to `=== null` to keep consistent with the listDiff inputs taking null or string.